src/components/Home/components/Header/index.js

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import classNames from "classnames";
```

Comment:
classNames is not a component. It's rather a utility. Please move it below FontAwesome